File: java/two-fer/src/main/java/Twofer.java
class Twofer {
    String twofer(String name) {
        // Put your code here.
    	if ((name == null) || (name.trim().length() == 0)) {
    		return "One for you, one for me.";
    	}
    	else {
    		return String.format("One for %s, one for me.", name);
    	}
    }
}

File: ruby/two-fer/two_fer.rb
=begin
Write your code for the 'Two Fer' exercise in this file. Make the tests in
`two_fer_test.rb` pass.

To get started with TDD, see the `README.md` file in your
`ruby/two-fer` directory.
=end
class TwoFer
    def self.two_fer(name = "")
        if name.strip().length == 0
            user = "you"
        else
            user = name.strip()
        end
        "One for #{user}, one for me."
    end
end

